x/cash/controller.go

```diff
@@ -64,10 +64,10 @@ func (c BaseController) MoveCoins(store weave.KVStore,
 		return err
 	}
 	if sender == nil {
-		return ErrEmptyAccount(src)
+		return errors.ErrEmpty.Newf("empty account %#v", src)
```

Comment:
I think %s will work well, if src is an Address, there is a String() method that returns hex.

Just remember not to use %X for Addresses, or it will be double-hex-encoded
(I have done this a few times and wondered why it is so long)